app/decorators/guest_decorator.rb

```diff
@@ -0,0 +1,13 @@
+class GuestDecorator
+  def lockout_time_remaining_in_words
```

Comment:
creating the generic message feels slightly out of scope for this PR but definitely something we should do so I created an issue and assigned myself 🏁